docs/reference/quickstart/getting-started.asciidoc

```diff
@@ -1,51 +1,90 @@
 [[getting-started]]
-== Quick start: Add data using Elasticsearch APIs
+== Quick start: Indices, documents, and mappings
 ++++
-<titleabbrev>Basics: Add data using APIs</titleabbrev>
+<titleabbrev>Basics: Indices, documents, and mappings</titleabbrev>
 ++++
 
-In this quick start guide, you'll learn how to do the following tasks:
+This quick start guide is a hands-on introduction to the fundamental concepts of Elasticsearch: indices, documents, and field type mappings. 
+You'll learn how to create an index, add data as documents, work with dynamic and explicit mappings, and perform your first basic searches.
 
-* Add a small, non-timestamped dataset to {es} using Elasticsearch REST APIs.
-* Run basic searches.
+[TIP]
+====
+By default the examples are in Dev Tools Console syntax, but you can {kibana-ref}/console-kibana.html#import-export-console-requests[export to other programming languages] in the Console UI.
```

Comment:
- The docs examples should be translated into other PL's in our docs code widget, so 'only available' isn't accurate, I just wasn't saying straight up "also in Python, Ruby, and JS" because that conversion process isn't immediate and doesn't have 100% coverage yet. So I don't want to say "sometimes available in those languages too".

- Also:
  - They will always be available in at least Console + curl 
  - The widget does display them in Console _by default_